packages/vite/src/node/server/index.ts

```diff
@@ -433,14 +433,15 @@ export async function _createServer(
       }
       return server
     },
-    openBrowser() {
+    openBrowser(open?: string) {
```

Comment:
Maybe renaming the param as `url` better represents the value here. The feature sounds good to add for me though.